Wireframe/index.html

```diff
@@ -4,10 +4,50 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <link rel="stylesheet" href="./css/style.css">
+
+    <header>
+      <h1>Git</h1>
+      <p>A Version Control Tool</p>
+    </header>
+  
+<body>
+
+  <main>
+    <div class="article">
```

Comment:
Is there a grouping element you could use that isn't a div? What difference does it make, using div vs article or section?